packages/components/Button/package.json

```diff
@@ -30,6 +30,7 @@
     "@fluentui-react-native/adapters": ">=0.8.5 <1.0.0",
     "@fluentui-react-native/experimental-activity-indicator": "^0.5.29",
     "@fluentui-react-native/experimental-text": ">=0.9.0 <1.0.0",
+    "@fluentui-react-native/experimental-shadow": "0.2.0",
```

Comment:
@rurikoaraki We add this dependency here because the `shadowStyleFromTheme` function used to exist in the Button package, but now we're moving it to the Shadow package. Do you see any issue with adding a new dependency here now that we're V1? If so, we can just duplicate the file, and then update Button when the Shadow work is complete.

Reply:
Adding dependencies is fine if it's necessary for new functionality.

Reply:
Ended up just duplicating the file